docs/website/docs/tutorial/sql_database.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Load data from a SQL database
+description: How to extract data from a SQL Database using dlt's SQL Database core source
+keywords: [sql connector, sql database pipeline, sql database]
+---
+
+This tutorial will show you how you can use `dlt` to load data from a SQL Database (PostgreSQL, MySQL, Microsoft SQL Server, Oracle, IBM DB2, etc.) into any `dlt`-compatible destination (Postgres, BigQuery, Snowflake, DuckDB, etc.).  
```

Comment:
```suggestion
This tutorial will show you how you can use dlt to load data from a SQL Database (PostgreSQL, MySQL, Microsoft SQL Server, Oracle, IBM DB2, etc.) into any dlt-compatible destination (Postgres, BigQuery, Snowflake, DuckDB, etc.).  
```
Removed backticks since in this context we're talking about dlt as project and not as an actual command do execute.